src/utils.ts

```diff
@@ -1,3 +1,20 @@
 export function isAccountPrivileged(sender: string): boolean {
   return sender.endsWith(":parity.io") || sender.endsWith(":web3.foundation");
 }
+
+export const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Predicate = () => any | (() => Promise<any>);
+
+export const waitUntil = async (predicate: Predicate, step = 100, timeout = 10000) => {
```

Comment:
Also, the timeout here's the same as the governing timeout for the tests, meaning that this timeout condition will never fire, and instead we'll be getting timeouts from jest, without stacktrace

Reply:
Good catch, timeouts adjusted.